### ISSUE TITLE
Does not work without PDF Invoice

### ISSUE THREAD

**Issue by alex-james** [mageplaza/magento-2-email-attachments#28]
This module does not work without the PDF Invoice module and the instructions do not make this clear, just spent an hour installing, configuring, testing and debugging this module to realise that the code required to create the PDF files is not present in this module.

Please update the guidance to make this clear so other people do not waste their time with this module.

**Comment by Shinichi69** [comment]
Hi @alex-james 

I’m Shin from Mageplaza and I will support you on this issue.

Our Email Attachment module can work independently without the PDF Invoice module.
Could you provide me steps to reproduce your problem?

Thanks so much!

**Comment by alex-james** [comment]
Hi Shin,

Thanks for coming back to me, whilst the module doesn't "fail" it doesn't appear to do anything without the PDF Invoice module, I configured as per your instructions, enabled the module and ran setup:upgrade but all my order emails were without any PDF attachments. When I looked at the code, the sending of attachments and indeed the attachments themselves seem to be dependant on config settings that don't exist in this module and don't come from this module. Whilst it adds the relevant hooks into the Magento email system and I can see them being triggered, it essentially does not nothing from what I can tell.

Kind regards

Alex
